app/src/main/res/values/strings.xml

```diff
@@ -1206,6 +1206,11 @@ Alternatively, you can leave a note (with a photo)."</string>
     <string name="achievement_lifesaver_solved_X">You solved %d quests that help emergency services and first responders!</string>
     <string name="quest_airConditioning_no_name_title">"Does this place (%s) have air conditioning?"</string>
     <string name="quest_airConditioning_name_type_title">"Does %1$s (%2$s) have air conditioning?"</string>
+    <string name="quest_smoking_no_name_title">"Is smoking allowed anywhere in this place (%s)?"</string>
+    <string name="quest_smoking_name_type_title">"Is smoking allowed anywhere in %1$s (%2$s)?"</string>
+    <string name="quest_smoking_outside">Outside only</string>
+    <string name="quest_smoking_isolated">Prohibited everywhere, except in separate completely isolated smoking room</string>
```

Comment:
```suggestion
    <string name="quest_smoking_isolated">Prohibited everywhere, except in a separate, isolated smoking room</string>
```

Just some grammar shenanigans.

Reply:
Thanks, the language was awkward in my attempt to _**OVEREMPHASIZE**_ that part on purpose. Obviously it does not work as well as I hoped :smile:, see https://github.com/streetcomplete/StreetComplete/pull/3784#discussion_r815350764